tests/lax_numpy_test.py

```diff
@@ -6104,13 +6104,8 @@ def testWrappedSignaturesMatch(self):
     self.assertEqual(mismatches, {})
 
 
-_all_dtypes: List[str] = [
-  "bool_",
-  "uint8", "uint16", "uint32", "uint64",
-  "int8", "int16", "int32", "int64",
-  "float16", "float32", "float64",
-  "complex64", "complex128",
-]
+_available_numpy_dtypes: List[str] = [dtype.__name__ for dtype in jtu.dtypes.all
+                                      if dtype != dtypes.bfloat16]
```

Comment:
Note: `jtu.dtypes.all` contains the supported dtypes logic, which is why that check can be deleted in the test function below.